runtime/src/accounts.rs

```diff
@@ -278,26 +276,48 @@ impl Accounts {
                             })
                             .unwrap_or_default();
 
-                        if account.executable() && bpf_loader_upgradeable::check_id(account.owner())
```

Comment:
Looks like the owner check was removed in order to apply this write restriction to all executable accounts. I think it would be better to distinguish between non-upgradeable programs and upgradeable programs though to make it impossible to write lock the token program.

Reply:
Oh, only include the `is_upgradeable_loader_present` exception if the owner is the upgradeable loader? Yeah, makes sense